[Email Routing] New UI (#4243)
cloudflare/cloudflare-docs: content/email-routing/get-started/email-addresses.md

---

> title: Configure rules and addresses

Reviewer: This page seems more of a reference for the Routes page than a getting started step...
Probably for later: Consider having specific content for the getting started, where you could link to specific sections in this page.

pcx-content-type: how-to
weight: 2
---

# Routes

The Routes page is divided into three main configuration sections:

- Custom addresses
- Catch-all address
- Destination addresses

Each section lets you configure different aspects of Email Routing.

{{<Aside type="note" header="Note">}}

> A custom address and destination address form a **rule**. You can enable, disable and delete rules.

Reviewer: ```suggestion
A custom address and destination address form a rule. You can enable, disable, and delete rules.
```

{{</Aside>}}

## Custom addresses

In this section you can create, enable, disable, and edit custom addresses. To add new custom addresses:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account and domain.
2. Go to **Email** > **Routes**.
3. Click **Create address**.

> 4. In **Custom address**, enter the custom email address you want to use (for example, `my-new-email`).

Reviewer: ```suggestion
4. In **Custom address**, enter the custom email address you want to use (for example, `my-new-email@example.com`).
```
I'd include a domain so that the expected format is clear.

Reviewer: Makes sense! It's harder when we're not seeing the UI, I guess.

5. In the **Action** drop-down menu, choose what this rule should do. You can choose between _Send to_ or _Drop_. Refer to [Rules actions](#rules-actions) for more information.
6. In **Destination address**, choose the email address you want your emails to be forwarded to — for example, `your-name@gmail.com`. You can only choose from destination addresses you have already verified. To add new destination addresses, refer to [Destination addresses](#destination-addresses).

{{<Aside type="note">}}
If you have more than one destination address linked to the same custom address, Email Routing will only process the most recent rule. This means only the most recent pair of custom address and destination address (rule) will receive your forwarded emails. To avoid this, do not link more than one destination address to the same custom address.
{{</Aside>}}

> ### Rules actions

Reviewer: I'd write "**Rule** actions" here (personal preference, maybe).

When creating a rule, you must specify an **Action**:

- _Send to_: Emails will be routed to your **Destination address**. This is the default action.

> - _Drop_: Deletes email sent to the **Custom address** without routing them. This can be useful if you want to make an email address appear valid for privacy reasons.

Reviewer: ```suggestion
- _Drop_: Deletes emails sent to the **Custom address** without routing them. This can be useful if you want to make an email address appear valid for privacy reasons.
```

{{<Aside type="note">}}

> All rules are automatically disabled until the destination address is validated by the user. This is to prevent spamming unintended recipients.

Reviewer: ```suggestion
To prevent spamming unintended recipients, all rules are automatically disabled until the destination address is validated by the user.
```

{{</Aside>}}

### Disable an email rule

> Email rules are pairs of custom addresses and destination addresses. To disable a rule:
> 

Reviewer: ```suggestion
```
I'd avoid repeating the definition of rule, and the second sentence would become redundant on its own.

Maybe in the definition (above) you could also use the term "email rule" instead of just "rule"? Also, a bit below you call them "Email Routing rules" — I'd select at most two terms out of "rule", "email rule", "Email Routing rule", and "routing rule" instead of using all of these.

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account and domain.
2. Go to **Email** > **Routes**.
3. In **Custom addresses**, identify the rule you want to pause, and toggle the status button to **Disabled**.

> Your email rule is now disabled and will not forward emails to a destination address. You will have to toggle the status button to **Active** for this rule to forward emails again.

Reviewer: ```suggestion
Your email rule is now disabled and will not forward emails to a destination address. To forward emails again, toggle the rule status button to **Active**.
```

### Edit custom addresses

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account and domain.
2. Go to **Email** > **Routes**.
3. In **Custom addresses**, identify the rule you want to edit, and click **Edit**.
4. Make the appropriate changes to this custom address.

## Catch-all address

> When you toggle this feature on, it enables Cloudflare to catch variations of email addresses to make them valid for the specified domain. For example, if you created a rule for `info@yourdomain.com` and a sender accidentally types `ifno@yourdomain.com`, the email will still be correctly handled if you have **Catch-all** addresses enabled.

Reviewer: ```suggestion
When you enable this feature, Cloudflare will catch variations of email addresses to make them valid for the specified domain. For example, if you created a rule for `info@example.com` and a sender accidentally types `ifno@example.com`, the email will still be correctly handled if you have **Catch-all** addresses enabled.
```
